### PR TITLE
better description of group property

### PR DESCRIPTION
**Pull request by libor-peltan-cznic**

### CODE REVIEW

**Comment by Habbie** [comment]
Text makes sense to me. We should write something better than 'respective PTR record name'.

Can you format to one sentence per line, like the rest of the file?

**Comment by Habbie** [comment]
Oh, and we should make extra clear that this property is optional to support.

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 1]
```diff
@@ -213,15 +213,17 @@ querying via recursive resolvers.
 
 # Properties {#properties}
 
-Member zones may optionally be assigned additional properties represented by
-RRsets below the corresponding member node. This document suggests a few
-properties which are all optional to implement.
+Each member zone MAY have one or more of additional properties, described in this chapter.
+These properties are completely optional and the catalog zone consumer SHOULD ignore those it doesn't understand.
```
Comment (Habbie) [inline draft-ietf-dnsop-dns-catalog-zones.md:217]:
```suggestion
Each member zone MAY have one or more additional properties, described in this chapter.
These properties are completely optional and the catalog zone consumer SHOULD ignore those it does not understand.
```

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 2]
```diff
@@ -213,15 +213,17 @@ querying via recursive resolvers.
...
+Each property is defined by a zone record in a sub-node of the node, in which the member-defining PTR record resides.
 
 ## The Group Property
 
-The group property allows to add at most one tag per member zone.
-The consumer might handle/configure member zones differently based on the tag.
-The group property is represented by a TXT Resource Record, for example:
+The property is defined by a TXT record in the sub-node labelled `group`.
+The value of the TXT record SHOULD be at most 255 octets long and SHOULD NOT contain whitespace characters.
```
Comment (Habbie) [inline draft-ietf-dnsop-dns-catalog-zones.md:223]: I'm fine with making this MUST, both times.

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 3]
```diff
@@ -213,15 +213,17 @@ querying via recursive resolvers.
...
+The consumer SHOULD interpret the value case-sensitively.
```
Comment (Habbie) [inline draft-ietf-dnsop-dns-catalog-zones.md:224]: also MUST, if you ask me

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 4]
```diff
@@ -213,15 +213,17 @@ querying via recursive resolvers.
 
 # Properties {#properties}
 
-Member zones may optionally be assigned additional properties represented by
-RRsets below the corresponding member node. This document suggests a few
-properties which are all optional to implement.
+Each member zone MAY have one or more of additional properties, described in this chapter.
+These properties are completely optional and the catalog zone consumer SHOULD ignore those it doesn't understand.
+Each property is defined by a zone record in a sub-node of the node, in which the member-defining PTR record resides.
```
Comment (Habbie) [inline draft-ietf-dnsop-dns-catalog-zones.md:218]: I preferred the original text, so:

```suggestion
Properties are represented by RRsets below the corresponding member node.
```

**Review by Habbie — CHANGES_REQUESTED** [review]

**Review by Habbie — APPROVED** [review]

**Review by wtoorop — APPROVED** [review]
Perfect!

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 4 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md]: Resolved by the last commit

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 3 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md]: Resolved in last commit

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 2 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md]: Resolverd

**Pull request merged**